spec/integration/aws_lambda_examples_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+require "spec_helper"
+
+describe "AWS Lambda examples in Ruby" do
+  require_aws_auth
+
+  it "shares the client" do
+    # Start AWS Lambda Example 1
+
+    # Import the dependency.
+    require "mongo"
+
+    # Create a Mongo::Client instance.
+    # CRITICAL: You must create client outside the handler so that the client
+    # can be reused across function invocations.
+    client = Mongo::Client.new(ENV["MONGODB_URI"])
+
+    def lambda_handler(event:, context:)
+      # Use the client to return the name of the connected database.
```

Comment:
Suggest "configured database", since this will return regardless of whether the connections succeed (and even when the client is constructed with an empty host list).

Reply:
Or actually perform an operation which would perhaps be more meaningful.